### DISCUSSION TITLE
下载和导出审计日志 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1050] [Announcements]
# 下载和导出审计日志 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/user-guide/audit-log/

**Comment by liyuerich** [comment]
导出 是按照不同的查询条件导出吗？ 还是全部导出？

**Reply by rokkiter** [reply]
按照查询条件导出

**Comment by windsonsea** [comment]
目前只能导出某个时间范围的日志：

<img width="1263" alt="image" src="https://user-images.githubusercontent.com/79828097/226509811-b1ccb1d5-573e-4871-935c-677e6d7731e7.png">
